zk/src/archive/web/js/zk/fmt/numfmt.js

```diff
@@ -170,6 +170,12 @@ zk.fmt.Number = {
 		if (val == null) return '';
 		if (!fmt) return val + '';
 		
+		if ('en-IN' == fmt) {
+			var dotIndex = val.indexOf('.'),
+				scale = dotIndex == -1 ? 0 : val.length - dotIndex - 1;
+			return new Intl.NumberFormat(fmt, {maximumFractionDigits: scale}).format(parseFloat(val));
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat, Intl.NumberFormat only supports IE11.

We might need a polyfill for IE9/10 like https://github.com/andyearnshaw/Intl.js

Reply:
ok